beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -112,7 +112,7 @@ func (s *Service) processFetchedData(
 
 	// Use Batch Block Verify to process and verify batches directly.
 	if err := s.processBatchedBlocks(ctx, genesis, data.blocks, s.chain.ReceiveBlockBatch); err != nil {
-		log.WithError(err).Debug("Batch is not processed")
+		log.WithError(err).Warn("Batch is not processed")
```

Comment:
can you remove `WithError` it prints the whole stack trace which can get very noisy. We just want to print the main error